versioned/spi/src/main/java/org/projectnessie/versioned/TracingVersionStore.java

```diff
@@ -138,11 +147,12 @@ public MergeResult<Commit> transplant(
     return TracingVersionStore
         .<MergeResult<Commit>, ReferenceNotFoundException, ReferenceConflictException>
             callWithTwoExceptions(
```

Comment:
I wonder whether we could use simple delegation code here with method-level `@WithSpan` annotations and instrument the bytecode with OTel at build time... WDYT?

Span attributes can be added via `Span.current().setAttribute(...)`

Reply:
In Quarkus, probably better, yea. Let's get that done for Quarkus.

Reply:
AFAIK, instrumentation can be done in a generic way with Quarkus providing runtime OTel impl.

Reply:
Yep, there's a way to do that.